lib/Driver/Driver.cpp

```diff
@@ -2003,8 +2003,15 @@ bool Driver::handleImmediateArgs(const ArgList &Args, const ToolChain &TC) {
     SuppressNoInputFilesError = true;
   }
 
-  if (const Arg *A = Args.getLastArg(options::OPT_driver_use_frontend_path))
-    DriverExecutable = A->getValue();
+  if (Args.hasArg(options::OPT_driver_use_frontend_path)) {
+    std::string commandString =
+        Args.getLastArgValue(options::OPT_driver_use_frontend_path);
```

Comment:
Not sure why you changed this from using `getLastArg`. (It's going to do two searches now…not that we're particularly efficient about that right now.)